electron/src/sso/SingleSignOn.test.main.ts

```diff
@@ -0,0 +1,10 @@
+import * as assert from "assert";
```

Comment:
Missing license header
and
```suggestion
import * as assert from 'assert';
```